test/benchmark_http_client_test.cc

```diff
@@ -179,7 +179,7 @@ class BenchmarkClientHttpTest : public Test {
   void setupBenchmarkClient(const RequestGenerator& request_generator) {
     client_ = std::make_unique<Client::BenchmarkClientHttpImpl>(
         *api_, *dispatcher_, store_, statistic_, false, cluster_manager_, http_tracer_, "benchmark",
-        request_generator, true);
+        request_generator, true, "");
```

Comment:
This is a lot of arguments here.  Would it be okay if we added comments helping users track the corresponding parameter under use?  For the new one, it'd be
```/*response_header_with_latency_input=*/""```
And then, the two boolean parameters could also benefit from this.